design-system/materials/internals/definition.yaml

```diff
@@ -220,12 +220,6 @@ choiceGroupsByTheme:
         break-point-jumbodesktop: 1920px
 
   test:
-    colors:
```

Comment:
This is unrelated but it's not needed as it is the same value as in the default theme.